sycl/include/CL/sycl/stream.hpp

```diff
@@ -327,6 +383,9 @@ inline const stream &operator<<(const stream &Out,
   switch (RHS) {
   case stream_manipulator::endl:
     Out << '\n';
+  case stream_manipulator::flush:
```

Comment:
Should this have a LLVM_FALLTHROUGH or similar?

Reply:
Thx, fixed!